src/components/comments/CommentComponent.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable react/prop-types */
+import React, { Component } from 'react';
+import Form from '../common/formContainer';
+import authorImage from '../../assets/images/profile_plaholder.png';
+
+export class CommentComponent extends Component {
+  render() {
```

Comment:
@patrickngabo, we agreed that we have to use functional components and pass props from the parent component.
but I can see that in this file you are not following it why?